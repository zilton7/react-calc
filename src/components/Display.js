import React from "react";

function Display({ calculation }) {
  return (
    <React.Fragment>
      <input id="result" value={calculation || "0"}></input>
    </React.Fragment>
  );
}

export default Display;
