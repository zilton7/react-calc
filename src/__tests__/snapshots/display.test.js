import renderer from 'react-test-renderer';
import Display from '../../components/Display';

it('renders Display correctly', () => {
  const tree = renderer.create(<Display />).toJSON();
  expect(tree).toMatchSnapshot();
});
