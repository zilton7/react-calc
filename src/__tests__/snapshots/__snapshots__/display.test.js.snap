// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Display correctly 1`] = `
<div
  className="display"
>
  <input
    defaultValue="0"
    id="result"
  />
</div>
`;
